.github/actions/localize-push-pull/action.yml

```
@@ -38,4 +41,5 @@ runs:
     - ${{ inputs.action }}
     - ${{ inputs.type }}
     - ${{ inputs.languages }}
-    - ${{ inputs.filepaths }}
\ No newline at end of file
+    - ${{ inputs.input-path }}
+    - ${{ inputs.output-path}}
```

Doubt the spacing here matters but just in case
```suggestion
    - ${{ inputs.output-path }}
```

---

ah thank you.